### PR TITLE
What it's like to interview

### PR DESCRIPTION
**Pull request by softwaregravy**

### CODE REVIEW

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+comments: true
+categories: 
+author: John Hinnegan
+---
+
+
+We are hiring like crazy here at Thinknear. 
+We're solving massive scale challenges in the hundreds of thousands of requests per second, pressing databases to the limit, and we have more data than we know what to do with. 
+As a result, we're looking for engineers, data scientists, and managers. 
+All these people are going through our process, and I wanted tos hare a bit more about that.
```
Comment (eduardog) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: s/tos hare/to share/

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+I'm going to skip the part where candidates first hear about us. 
+We connect with people through recruiters, linked in, personal networks, events hosted at our offices, on campuses, .... so it's different for everyone.
+
+The first step in our process for junior and mid-level candidates is to complete a coding challenge. 
+This is an online coding test, the goal of which is to provide a coding sample to an engineer to review before doing a phone screen.
+This step exists because we get a lot of people applying for positions who can't code.
+It's a bit frustrating, but this step helps save time for both parties -- since the position here requires writing lots of code, we really want to make sure up front that we cover it.
+
+Very sr. candidates might opt for what we call a 'sales call'.
```
Comment (igorb2) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: might opt?  this sounds like it's a candidate choice.  it's usually not.  unless they came from a well known recruiter.

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Simple problems involve sorting arrays, traversing trees, hashing strings, counting, and other problems you might encounter in a CS undergrad education.
+Our problems generally have multiple solutions, and though we'd like to see you produce the optimal solution, that's not necessarily the primary evaluation criteria.
+We're really looking for how you approach the problem, the quality of the code you produce, how you verify your solution is correct, and how you analyze it's performance.
+
+We vary between 1 and 2 phone screens. 
+After that, we ask candidates to come meet us in person. 
+We will pay for travel for non-local candidates (and if later make an offer, we also cover relocation).
+
+For the in house, we ask that candidates wear comfortable clothes akin to what they'd wear to work on a regular basis.
+You won't wear a suit here on a regular basis, there's no point wearing one to your interview.
```
Comment (igorb2) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: delete 'on a regular basis' or replace it with 'unless it's Halloween'

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Just make sure you're clean and presentable; don't overdo the cologne or perfume if that's your thing.
```
Comment (igorb2) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: I don't think we want to get into personal hygiene on the blog.

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+I'm going to skip the part where candidates first hear about us. 
+We connect with people through recruiters, linked in, personal networks, events hosted at our offices, on campuses, .... so it's different for everyone.
+
+The first step in our process for junior and mid-level candidates is to complete a coding challenge. 
+This is an online coding test, the goal of which is to provide a coding sample to an engineer to review before doing a phone screen.
+This step exists because we get a lot of people applying for positions who can't code.
+It's a bit frustrating, but this step helps save time for both parties -- since the position here requires writing lots of code, we really want to make sure up front that we cover it.
+
+Very sr. candidates might opt for what we call a 'sales call'.
+This is a call between the candidate and a people manager where we discuss what the candidate is looking for and attempt to determine if Thinknear will be the right fit.
```
Comment (igorb2) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: may want to add that just like coding challenge this is designed to also designed to save our engineer's valuable time (as well as candidate's)

File: source/_posts/2015-02-24-interviewing-with-thinknear.markdown [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+We will pay for travel for non-local candidates (and if later make an offer, we also cover relocation).
+
+For the in house, we ask that candidates wear comfortable clothes akin to what they'd wear to work on a regular basis.
+You won't wear a suit here on a regular basis, there's no point wearing one to your interview.
+Just make sure you're clean and presentable; don't overdo the cologne or perfume if that's your thing. 
+
+The in house interviews vary a lot depending on the position and level. 
+Junior engineering candidates will tend to meet with more engineers and their interviews will focus very strongly on coding.
+Midlevel engineers will have product folks added to the loop and spend some time on design.
+Senior engineers will do more design and talk more about leadership, team building, and architecture.
```
Comment (igorb2) [inline source/_posts/2015-02-24-interviewing-with-thinknear.markdown]: Do we want to mention that we'll be serving up lunch with potential teammates?  Judging culture fit?  or would it appear too judgmental?

**Comment by igorb2** [comment]
done with 1st pass

**Pull request merged**